clockworkpi_keypad.ino

```diff
@@ -59,21 +61,22 @@
 #define KEYPAD_START_SHIFT  KEY_NUM_PLUS
 #define KEYPAD_LIGHT1_SHIFT KEY_END
 #define KEYPAD_LIGHT2_SHIFT KEY_PAGE_DOWN
+#define KEYPAD_LIGHT3_SHIFT KEY_INSERT
 #define KEYPAD_LIGHT4_SHIFT KEY_PAGE_UP
 #define KEYPAD_LIGHT5_SHIFT KEY_HOME
 
-const int pins[KEY_NUM] = {3, 6, 7, 8, 9, 10, 11, 12, 13, 0, 2, 3, 4, 6, 7};
-const int shift_pins[SHIFT_KEY_NUM] = {1, 5};
+const int pins[KEY_NUM] = {3, 6, 7, 8, 9, 10, 11, 12, 13, 0, 2, 3, 4, 5, 6, 7};
+const int shift_pins[SHIFT_KEY_NUM] = {1};
```

Comment:
As its now 1 value, should be an int instead of a vector of int.

`cons int shift_pin = 1` for example or a #define to save bytes